.github/workflows/send-main-push-event.yml

```diff
@@ -0,0 +1,19 @@
+name: Trigger prisma-engines-builds run
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  send-tag-event:
+    runs-on: ubuntu-22.04
+    steps:
+      - run: echo "Sending event for commit $GITHUB_SHA"
+      - name: Workflow dispatch to prisma/prisma-engines-builds
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: .github/workflows/build-engines.yml
+          repo: prisma/prisma-engines-builds
+          token: ${{ secrets.BOT_TOKEN }}
+          inputs: '{ "commit": "${{ github.event.client_payload.commit }}" }'
```

Comment:
```suggestion
          inputs: '{ "commit": "${{ github.sha }}" }'
```

Reply:
Thanks for catching that, that was a copy/paste mistake from https://github.com/prisma/engines-wrapper/blob/8c6c6e068b5c00d910e95dc98963029be0507351/.github/workflows/publish-engines.yml